scss/bitstyles/utilities/font/_.scss

```diff
@@ -0,0 +1,23 @@
+.#{$bitstyles-namespace}u-font--light {
+  font-weight: $bitstyles-font-weight-light;
+}
+
+.#{$bitstyles-namespace}u-font--normal {
+  font-weight: $bitstyles-font-weight-normal;
+}
+
+.#{$bitstyles-namespace}u-font--medium {
+  font-weight: $bitstyles-font-weight-medium;
+}
+
+.#{$bitstyles-namespace}u-font--bold {
+  font-weight: $bitstyles-font-weight-bold;
+}
+
+.#{$bitstyles-namespace}u-text--italic {
+  font-style: italic;
+}
+
+.#{$bitstyles-namespace}u-text--not-italic {
```

Comment:
Maybe `u-text--normal` is a better name here?

Reply:
Thanks! I’ll look into this and the mixup further down 🎉

Reply:
Ah, using `normal` as the variant wasn’t possible because it’s already used for the font-weight class further up in this file — in CSS, `font-weight` can be `normal`, and so can `font-style`. In the interests of keeping the classnames shorter, it seemed better to use `italic`/`not-italic` instead of adding `-style`/`-weight` to each of them